packages/design-system/src/styles/components/_Badge.scss

```diff
@@ -23,17 +45,29 @@ $badge-font-size-big: $base-font-size;
 
 .ds-c-badge--info {
   background-color: $color-primary;
+
+  @include badge-high-contrast-mode-remove-color;
```

Comment:
Is this mixin included in each badge variation to increase the specificity of background-color styling?

Asking because, and I maybe reading this wrong, aren't the levels of specificity equal in this line as in line 39?

Reply:
This is to enforce a HCM theme color in place of a color that might not be visible, and then signal to HCM that it shouldn't use a [backplate](https://www.gwhitworth.com/posts/2019/high-contrast-text-backplate/). 

I wrote this mixin mainly for DRY purposes, but we could also accomplish the same thing with the following selector:

``` css
.ds-c-badge,
.ds-c-badge--info,
.ds-c-badge--success,
.ds-c-badge--warn
.ds-c-badge--alert { … }
```